techniques/client-side-script/SCR16.html

```diff
@@ -1,79 +1,87 @@
-	// set timer for 4 min 40 sec, then ask user to confirm.
-	setTimeout('userCheck()', 280000);
-}
-function userCheck() {
-	// set page refresh for 20 sec
-	var id=setTimeout('pageReload()', 20000);
-	// If user selects "OK" the timer is reset 
-	// else the page will refresh from the server.
-	if (confirm("This page is set to refresh in 20 seconds. 
-	Would you like more time?"))
-	{
-	clearTimeout(id);
-	timeControl();
-	}
-}
-function pageReload() {
-	window.location.reload(true);
-}
-timeControl();
---&gt;
-&lt;/script&gt;
+  &lt;meta charset="utf-8"&gt;
+  &lt;title&gt;Stock Market Quotes&lt;/title&gt;
+  &lt;script&gt;
+    function timeControl() {
+      // set timer for 4 min 40 sec, then ask user to confirm.
+      setTimeout('userCheck()', 280000);
+    }
+    function userCheck() {
+      // set page refresh for 20 sec
+      var id=setTimeout('pageReload()', 20000);
+      // If user selects "OK" the timer is reset 
+      // else the page will refresh from the server.
+      if (confirm("This page is set to refresh in 20 seconds. 
+        Would you like more time?"))
+        {
+          clearTimeout(id);
+          timeControl();
+        }
+    }
+    function pageReload() {
+      window.location.reload(true);
+    }
+    timeControl();
+  &lt;/script&gt;
 &lt;/head&gt;
 &lt;body&gt;
-&lt;h1&gt;Stock Market Quotes&lt;/h1&gt;
-...etc...
+  &lt;h1&gt;Stock Market Quotes&lt;/h1&gt;
+  ...
 &lt;/body&gt;
-&lt;/html&gt;
-</pre>
+&lt;/html&gt;</pre>
       </section>
-   </section><section id="tests"><h2>Tests</h2>
+   </section>
+   <section id="tests">
+      <h2>Tests</h2>
       <section class="procedure"><h3>Procedure</h3>
-         <p>
-On a Web page that has a time limit controlled by a script: </p>
+         <p>On a Web page that has a time limit controlled by a script:</p>
          <ol>
-            <li>
-load the page and start a timer that is 20 seconds less than the time limit.
-    </li>
-            <li>
-when the timer expires, check that a confirmation dialog is displayed warning of the impending time limit.
-    </li>
+            <li>Load the page and start a timer that is 20 seconds less than the time limit.</li>
+            <li>When the timer expires, check that a confirmation dialog is displayed warning of the impending time limit.</li>
          </ol>
       </section>
-      <section class="results"><h3>Expected Results</h3>
+      <section class="results">
+         <h3>Expected Results</h3>
          <ul>
-            <li>
-#2 is true.
-    </li>
+            <li>#2 is true.</li>
          </ul>
       </section>
-   </section><section id="related"><h2>Related Techniques</h2><ul>
-      <li><a href="../client-side-script/SCR1">SCR1</a></li>
-   </ul></section></body></html>
\ No newline at end of file
+   </section>
+   <section id="related">
+      <h2>Related Techniques</h2>
+      <ul>
+         <li><a href="../client-side-script/SCR1">SCR1</a></li>
+      </ul>
+   </section>
+</body>
+</html>
```

Comment:
Is there supposed to be a newline at the end of the file?  There's an icon here indicating one is missing.

Reply:
I've tested all the pages locally and it's not made a difference whether there's a new line or not. There's probably something in the HTML spec about how whitespace is handled, but this [Mozilla white-space document](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Whitespace) popped up first.